versioned_docs/version-0.2.0/getting-started-with-cloud-clusters/prerequisites/prerequisites.mdx

```diff
@@ -65,14 +63,12 @@ Open the UDP Ports required for the KubeSlice Gateway Node in both public and pr
 
 ## Required Cluster Roles (RBAC) 
 :::caution
-Only the cluster administrator can install the KubeSlice Controller and
-the Slice Operator(Worker Operator).
+Only the cluster administrator can install the KubeSlice Controller and the Slice Operator(Worker Operator).
```

Comment:
Add a space between Slice Operator and (Worker Operator). Now, there is no space.

Reply:
@uma-kt  - you haven't responded to this comment. 

But I have verified the change.

Reply:
Updated,